features/sapmachine/update-sapmachine.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+import json
+import argparse
+
+from urllib.request import urlopen
+
+sapmachine_releases_json_url = 'https://sap.github.io/SapMachine/assets/data/sapmachine-releases-website.json'
+
+
+def loadReleases():
+    with urlopen(sapmachine_releases_json_url) as response:
+        releases = json.loads(response.read().decode())
+        return releases
+
+
+def loadChecksum(url):
+    with urlopen(url) as checksum:
+        checksum_file_content: str = checksum.read().decode()
+        # Need to split because the file contains the checksum and the file name
+        # We only want the checksum which is the first element in the line
+        checksum = checksum_file_content.split(' ')[0]
+        return checksum
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--majorJreVersion', type=str)
+    args = parser.parse_args()
+    major_jre_version = args.majorJreVersion
+    releases = loadReleases()
+    tag: str = releases['assets'][major_jre_version]['releases'][0]['tag']
+    version = tag.split('-')[1]
+    downloadUrlSapMachineBinary_amd64: str = releases['assets'][major_jre_version]['releases'][0]['jre']['linux-x64']
+    downloadUrlSapMachineBinary_aarch64: str = releases['assets'][major_jre_version]['releases'][0]['jre']['linux-aarch64']
```

Comment:
I'm not sure about the 'releases' array here.. It seems to always only have a single entry which is the lastest minor-patch version, but if this assumption is false this might not give us the latest patch release here.. need to clarify if this is okay.